masterlist.yaml

```
@@ -2846,6 +2846,19 @@ plugins:
         itm: 1
         udr: 1
 
+###### Locations - Overhauls ######
+  - name: 'atlanticofficesf23.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/25401' ]
```

Trailing slash for consistency:

```suggestion
    url: [ 'https://www.nexusmods.com/fallout4/mods/25401/' ]
```